deployment/3_deployContracts.js

```diff
@@ -48,11 +48,11 @@ async function main() {
         'trustedAggregatorTimeout',
         'pendingStateTimeout',
         'forkID',
-        'supernets2Owner',
+        'cdkValidiumOwner',
         'timelockAddress',
         'minDelayTimelock',
         'salt',
-        'supernets2DeployerAddress',
+        'cdkValidium2DeployerAddress',
```

Comment:
nit: this can be renamed to from `cdkValidium2DeployerAddress` to `cdkValidiumDeployerAddress`